### PR TITLE
Support ranch 1.1, move to rebar3.

### PR DESCRIPTION
**Pull request by omarkj**

### CODE REVIEW

**Comment by omarkj** [comment]
Requesting review @ferd @evanmcc @tsloughter

File: rebar.config [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+{profiles, [
+            {test, [{deps,
+                     [
+                      {ranch, [], {git, "https://github.com/extend/ranch.git",
+                                   {tag, "1.1.0"}}}
+                     ]
+                    },
+
+                    {erl_opts, [debug_info,
+                                warnings_as_errors]}]
```
Comment (ferd) [inline rebar.config]: This doesn't need to be duplicated; the erl_opts are merged with the default ones, so this only duplicates the settings.

**Comment by ferd** [comment]
+1, except for comment.

**Comment by evanmcc** [comment]
+1, may need some various dep-shuffling to get hermes to use the right version of ranch

**Comment by ferd** [comment]
@evanmcc yeah. We should do it later; the vegur stuff needs to be off the ranch fork too, so we'll look at eventually bumping hermes+vegur+ranch+ranch_proxy_protocol, and hoping cowboyku holds up under these modifications.

**Pull request merged**